tokenizer_config.json: 100%|████████████████| 48.0/48.0 [00:00<00:00, 56.0kB/s]
C:\Users\karth\AppData\Local\Programs\Python\Python313\Lib\site-packages\huggingface_hub\file_download.py:143: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\Users\karth\.cache\huggingface\hub\models--bert-base-uncased. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.
To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
  warnings.warn(message)
vocab.txt: 100%|████████████████████████████| 232k/232k [00:00<00:00, 1.16MB/s]
tokenizer.json: 100%|███████████████████████| 466k/466k [00:00<00:00, 2.19MB/s]
config.json: 100%|█████████████████████████████| 570/570 [00:00<00:00, 616kB/s]
Traceback (most recent call last):
  File "D:\Finrapt\intent-classification using bert\train_bert.py", line 52, in <module>
    train_dataset = Dataset.from_dict({"text": train_texts, "labels": train_labels}).map(tokenize, batched=True)
                                                                                         ^^^^^^^^
NameError: name 'tokenize' is not defined. Did you mean: 'tokenizer'? Or did you forget to import 'tokenize'?
